src/many-ledger/src/storage/event.rs

```diff
@@ -125,11 +129,40 @@ pub struct LedgerIterator<'a> {
 }
 
 impl<'a> LedgerIterator<'a> {
-    pub fn all(merk: &'a merk::Merk) -> Self {
-        Self::scoped_by_id(merk, CborRange::default(), SortOrder::Indeterminate)
+    pub fn all_multisig(merk: &'a merk::Merk, order: SortOrder) -> Self {
+        use crate::storage::multisig::MULTISIG_TRANSACTIONS_ROOT;
+        use rocksdb::IteratorMode;
+
+        // Set the iterator bounds to iterate all multisig transactions.
+        // We will break the loop later if we can.
+        let mut options = ReadOptions::default();
+        options.set_iterate_lower_bound(MULTISIG_TRANSACTIONS_ROOT);
+
+        let mut bound = MULTISIG_TRANSACTIONS_ROOT.to_vec();
+        bound[MULTISIG_TRANSACTIONS_ROOT.len() - 1] += 1;
```

Comment:
I recall asking this question before but I don't get what this is supposed to do.

Why not just `options.set_iterate_range(MULTISIG_TRANSACTIONS_ROOT.to_vec()..)` or something?

Reply:
Done with something better even. Thanks for pointing me towards this, haven't seen this method.